packaging/macOS/README.md

```diff
@@ -93,12 +96,35 @@ $ rm dist/parsec.zip
 $ ditto -c -k --sequesterRsrc --keepParent "dist/parsec.app" dist/parsec.zip
 ```
 
-
-### 6 - Check with Gatekeeper
+#### Check with Gatekeeper
 
 Steps 4 and 5 exist to comply with Gatekeeper.
-We can verify if both steps were successful by asking Gatekeeper directly:
+We can check if both steps were successful by asking Gatekeeper directly:
 
 ```shell
 $ spctl --assess --type execute -vvv "dist/parsec.app"
 ```
+
+
+### 6 - Bundle into .dmg
+
+The app is now ready to be bundled for distribution. To this end, we create a drag & drop `.dmg` folder.
+A `.dmg` background is needed at this point, many neutral backgrounds can be found with a quick search, until we eventually design our own.
```

Comment:
Can't we just store a background image in our sources instead ? Doesn't seem like a good idea to have to search for a background image each time we want to do the packaging ;-)